.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Don't check in anything in the .idea folder. Fix this by commiting the removal of the folder. Then in a separate commit, add `.idea/` to the .gitignore file.